<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileSelectDialogue</class>
 <widget class="QDialog" name="FileSelectDialogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="FileSelect">
    <item>
     <layout class="QHBoxLayout" name="File">
      <item>
       <widget class="QLabel" name="L_FPTitle">
        <property name="text">
         <string>File Path* :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="L_FilePath">
        <property name="text">
         <string>&lt;FilePath&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="B_SelectFile">
      <property name="text">
       <string>Select File</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>82</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="Image">
    <item>
     <widget class="QLabel" name="L_Image">
      <property name="text">
       <string>Image</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="GV_CoverArt"/>
    </item>
    <item>
     <widget class="QPushButton" name="B_SelectImage">
      <property name="text">
       <string>Select Image</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>361</width>
     <height>82</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="SongInfo">
    <item>
     <layout class="QHBoxLayout" name="SongTitle">
      <item>
       <widget class="QLabel" name="L_SongTitle">
        <property name="text">
         <string>Song Title*</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_SongTitle"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="Artist">
      <item>
       <widget class="QLabel" name="L_Artist">
        <property name="text">
         <string>Artist</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>61</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_Artist"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="Album">
      <item>
       <widget class="QLabel" name="L_Album">
        <property name="text">
         <string>Album</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>57</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_Album"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
